08-components-lab/src/01-timeconverter.jsx

```diff
@@ -8,5 +8,49 @@
 
 import React from 'react';
 
-export default React.createClass({
-});
+export default class TimeConverter extends React.Component{
+    constructor(props){
+        super(props);
+        this.state={
+            number : 0
```

Comment:
instead of calling it a number give it more descriptive name. Think of the smallest value you use for computing seconds, minutes, hours and use it instead.